### ISSUE TITLE
buttercup needs active internet connection to accept password that unlocks the vault !

### ISSUE THREAD

**Issue by ihubgit** [buttercup/buttercup-desktop#1059]
hi, using BC as desktop app on MacOS. When using network blocking app I'm unable to unlock BC vaults anymore ! Getting the following error :

> Error invoking remote method 'unlock-source': Layerr: Failed unlocking vault source: Failed unlocking source: Authentication failed while decrypting content

After desactivating the network app, i can unlock the vault again, which means BC connects silently to the Internet even if i merely want to unlock a vault locally !

**Comment by perry-mitchell** [comment]
Buttercup has no internet-bound dependency besides checking for updates, which should fail silently if you have no connection. BC doesn't make any other requests **unless** your vault source is a remote one, like Dropbox for example.

What type of vault were you trying to unlock?

**Comment by ihubgit** [comment]
It was a local vault.

**Comment by perry-mitchell** [comment]
Local vaults don't require a network connection, so something else must have happened when you attempted to disable your network blocking application. I just tested this myself by turning off Wifi and opening a local vault - it opened fine.

I did see an update failed error, which should not be shown imo, so I'll address that. But I don't believe there's anything actionable here right now, so I'll close this issue.
